### PR TITLE
This week's stuff

### PR DESCRIPTION
**Pull request by Reisii**
Pls go easy on me

### CODE REVIEW

File: Tufgaben2.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+test1 = ['1', 'd2', 'g', 'asdf', '-12', '+234', '4l;d9', 'h']
+test2 = ['jalo', '456', 'g', '00', '-kell', '+=', '73812', 'h']
+test3 = ['0', 'd', '3', 'a', '-1', '+', 'l', 'h']
+
+
+def array_filter(arr):
+    char_list = []
+    integer_list = []
+    for i in range(0, len(arr)):
```
Comment (fabischw) [inline Tufgaben2.py:67]: Works here but just looping trough the array elements would have been easier and easier to understand.

File: Tufgaben2.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        if (len(arr[i]) == 1) and arr[i].isnumeric():
+            integer_list.append(arr[i])
+            char_list.append(arr[i])
+        elif arr[i].isnumeric():
+            integer_list.append(arr[i])
+        elif len(arr[i]) == 1:
+            char_list.append(arr[i])
```
Comment (fabischw) [inline Tufgaben2.py:74]: this solution works but using one if statement for the number matching and a second if case with same indentation for chars would have been easier.

**Review by fabischw — CHANGES_REQUESTED** [review]
Job well done; a few things could've been implemented with reduced complexity as highlighted by my comments. You may also consider using docstrings next time.
Great solution overall, source code changes are not necessarily required.
However, your python file seems to be in the parent directory  of the repo instead of a personal folder in ./Exercises. This issue has to be resolved.

**Comment by atrejut** [comment]
@Reisii Please try to get this old PR merged...

**Review by fabischw — APPROVED** [review]
Problems were adressed.

**Comment by fabischw** [comment]
I will be merging this PR if it does not get merged by the PR creator within the next 3 days in an effort to clean up the repo (More information can be found in [Issue #228](https://github.com/dhbw-ma-ppp/ppp-2023/issues/228))

**Pull request merged**